documentation/staging/content/userguide/base-images/patch-images.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Patch running domains"
+date: 2019-02-23T16:45:55-05:00
+weight: 3
+description: "Dynamically update WebLogic images in a running domain."
+---
+
+### Apply patched images to a running domain
+
+When updating the WebLogic binaries of a running domain in Kubernetes with a patched container image,
+the operator applies the update in a zero downtime fashion.
+The procedure for the operator to update the running domain differs depending on the
+[domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}).
+See the following corresponding sections:
+
+- [Domain in PV](#domain-in-pv)
+- [Model in Image with auxiliary images](#model-in-image-with-auxiliary-images)
+- [Model in Image without auxiliary images](#model-in-image-without-auxiliary-images)
+- [Domain in Image](#domain-in-image)
+
+For a broader description of managing the evolution and mutation
+of container images to run WebLogic Server in Kubernetes,
+see [CI/CD]({{< relref "/userguide/cicd/_index.md" >}}).
+
+#### Domain in PV
+
+{{% notice warning %}}
+Oracle strongly recommends strictly limiting access to Domain in PV domain home files.
+A WebLogic domain home has sensitive information
+including credentials that are used to access external resources
+(for example, a data source password),
+and decryption keys
+(for example, the `DOMAIN_HOME/security/SerializedSystemIni.dat` domain secret file).
+{{% /notice %}}
+
+For Domain in PV domains,
+the container image contains only the JDK and WebLogic Server binaries,
+and its domain home is located in a Persistent Volume (PV)
+where the domain home is generated by the user.
+
+For this domain home source type, you can create your own patched images using the steps
+in [Create a custom image with patches applied](#create-a-custom-image-with-patches-applied)
+or you can obtain patched images from the Oracle Container Registry,
+see [Obtain images from the Oracle Container Registry]({{< relref "/userguide/base-images/ocr-images#obtain-images-from-the-oracle-container-registry" >}}).
+
+To apply the patched image,
+edit the Domain Resource image reference with the new image name/tag
+(for example, `oracle/weblogic:12.2.1.4-patched`).
+The operator will then automatically perform a
+[rolling restart]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting#overview" >}})
+of the WebLogic domain to update the Oracle Home of the servers.
+For more information on server restarts,
+see [Restarting]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting.md" >}}).
+
+#### Model in Image with auxiliary images
+
+For Model in Image domains when using auxiliary images:
+
+- The container image contains only the JDK and WebLogic Server binaries.
+- The [WebLogic Deployment Tooling](https://oracle.github.io/weblogic-deploy-tooling/) (WDT) installation,
+  WDT model files, and application archive files,
```

Comment:
I think this wording is a little awkward.  If `WDT model files` only refer to the YAML/JSON model file, then you left out WDT variable files.  If `WDT model files` refer to all three WDT file types, then there is no need to call out the application archive separately.

Reply:
Added WDT variable files.